### PR TITLE
fix: make tests pass for current guides

### PR DESCRIPTION
**Pull request by JacquesDurand**
| Q             | A
| ------------- | ---
| Branch?       | docs
| Bug fix?      | no 
| New feature?  | no 
| Deprecations? | no
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | api-platform/docs#... 

Fixes current guides to make tests green.

### CODE REVIEW

File: docs/guide/using-doctrine-orm-filters.php [hunk 1]
```diff
@@ -239,31 +221,38 @@ final class BookFixtures extends Fixture
...
-            $janeDoe = AnonymousFactory::new(User::class)::createOne(['username' => 'jane.doe']);
-            AnonymousFactory::new(Book::class)::createMany(10, ['user' => $johnDoe]);
-            AnonymousFactory::new(Book::class)::createMany(10, ['user' => $janeDoe]);
+            $userFactory = AnonymousFactory::new(User::class);
+            $johnDoe = $userFactory->create(['username' => 'john.doe']);
+            $janeDoe = $userFactory->create(['username' => 'jane.doe']);
+
+            $bookFactory = AnonymousFactory::new(Book::class);
+            $bookFactory->many(10)->create([
+                'title' => 'title',
```
Comment (vincentchalamon) [inline docs/guide/using-doctrine-orm-filters.php:230]: Why such a fake name? Can't it be auto-faked by AnonymousFactory?

File: docs/guide/using-doctrine-orm-filters.php [hunk 2]
```diff
@@ -43,7 +43,6 @@ public function getId(): ?int
      * Each Book is related to a User, supposedly allowed to authenticate.
      */
     #[ApiResource]
-    #[ApiFilter(UserFilter::class)]
```
Comment (vincentchalamon) [inline docs/guide/using-doctrine-orm-filters.php:46]: Why disabling it?

File: docs/guide/using-doctrine-orm-filters.php [hunk 3]
```diff
@@ -186,24 +186,6 @@ function configure(ContainerConfigurator $configurator) {
      }
  }
 
-namespace App\Configurator {
```
Comment (vincentchalamon) [inline docs/guide/using-doctrine-orm-filters.php:189]: Same: why removing this configuration?

File: docs/guide/using-doctrine-orm-filters.php (hunk 2 above)
Comment (JacquesDurand) [reply docs/guide/using-doctrine-orm-filters.php:46]: This is not the goal of this guide, cf [this part in the doc](https://api-platform.com/docs/core/filters/#using-doctrine-orm-filters)

File: docs/guide/using-doctrine-orm-filters.php (hunk 3 above)
Comment (JacquesDurand) [reply docs/guide/using-doctrine-orm-filters.php:189]: The convention for configuration in guides is to namespace by `App\DependencyInjection` : cf [pdg application Kernel](https://github.com/JacquesDurand/php-documentation-generator/blob/main/tests/Application/src/Kernel.php#L96)

File: docs/guide/using-doctrine-orm-filters.php (hunk 1 above)
Comment (JacquesDurand) [reply docs/guide/using-doctrine-orm-filters.php:230]: I thought of using a variable since the factory is used more than once in this function ?

**Pull request merged**